scss/colors/synthesis-palette.scss

```
@@ -0,0 +1,36 @@
+// stoplight color system
+$synth-green-1-success: #015029;
+$synth-green-2-success: #09914f;
+$synth-green-3-success: #d4f6e5;
```

```suggestion
$synth-success-green-dark: #015029;
$synth-success-green-medium: #09914f;
$synth-success-green-light: #d4f6e5;
```

To me, this convention is easier to know what you're getting from just looking at the variable name.

---

I think we can keep this naming convention for now as it makes sense to me. As we talked about, as long as we have some primitives for these colors that get assigned to the semantic ones currently shown here we should be fine. We may end up adjusting or adding colors later to account for a sliding scale for brightness / a11y to make the system more logical on the primitive level.

`$synth-green-1` or `$synth-green-100`. I'm not too picky at this point, just as long as the primitive exists.

---

> I think we can keep this naming convention for now as it makes sense to me.

Keep the original naming convention or the one I suggested?

---

We can keep your suggested one!

---

Cool. Just wanted to make sure I was reading that correctly before I made changes.